pkg/gitreceive/build.go

```diff
@@ -41,7 +42,7 @@ func run(cmd *exec.Cmd) error {
 	return cmd.Run()
 }
 
-func build(conf *Config, s3Client *s3.S3, kubeClient *client.Client, builderKey, rawGitSha string) error {
+func build(conf *Config, s3Client *s3.S3, kubeClient *client.Client, env sys.Env, fs sys.FS, builderKey, rawGitSha string) error {
```

Comment:
This is ticky-tacky, but... all other functions that accept both a sys.FS and sys.ENV interface do so in according to an ordered arity that looks like `(<params>...sys.FS, sys.ENV,...<params>)`. Maybe switch the order of `env sys.Env` and `fs sys.FS` params in `build()` above.